src/components/inputs/z-select/index.tsx

```diff
@@ -126,6 +130,17 @@ export class ZSelect {
     });
   }
 
+  /** Emitted on reset selected item, returns select id, selected item id */
+  @Event()
+  resetSelect: EventEmitter;
+
+  private emitResetSelect(): void {
+    this.resetSelect.emit({
+      id: this.htmlid,
+      selected: "reset",
```

Comment:
Domanda, magari scema: serve questo campo? Non basta che le'vento sia di reset per individuarlo nel prodotto che lo userà? Questo evento vuol dire "reset", penserei sia sufficiente. No?

Reply:
troppa completezza che non serve in effetti, tolgo